contracts/src/utils/proxy/ProxyLib.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.8;
+
+import {ERC1967Proxy} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import {Clones} from "@openzeppelin/contracts/proxy/Clones.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+/// @title ProxyLib
```

Comment:
I created this factory to provide `deploy...Proxy` methods allowing for [testing without hardhat upgrades and warning silencing](https://github.com/aragon/osx/blob/e7ba46026db96931d3e4a585e8f30c585906e1fc/packages/contracts/test/test-utils/proxy.ts#L1-L29) and to [replace the free function `createERC1967Proxy` from `Proxy.sol`](https://github.com/aragon/osx/blob/e7ba46026db96931d3e4a585e8f30c585906e1fc/packages/contracts/src/utils/Proxy.sol#L7-L15). I could've also copied things over, but I opted against it as it would mean more work adapting the two repos afterward.